Resources/Locale/en-US/ui/options.ftl

```diff
@@ -95,6 +95,7 @@ ui-options-function-open-entity-spawn-window = Open entity spawn menu
 ui-options-function-open-sandbox-window = Open sandbox menu
 ui-options-function-open-tile-spawn-window = Open tile spawn menu
 ui-options-function-open-admin-menu = Open admin menu
+ui-options-function-lock-action-slots = (Un)lock dragging and clearing action slots
```

Comment:
this shouldn't be in options.ftl if it's not in the actual options menu, make a new file for it (if something like actionbar.ftl doesn't already exist)

Reply:
ok